target_postgres/postgres.py

```diff
@@ -125,6 +124,8 @@ def write_batch(self, stream_buffer):
                                                              (stream_buffer.stream,),
                                                              stream_buffer.stream)
 
+                table_metadata = self._get_table_metadata(cur,
```

Comment:
This should probably be moved to line ~140 as it's only used there. By doing that it better scopes the variable so that it's not being used accidentally in more places.